app/models/policy.rb

```diff
@@ -94,8 +94,23 @@ def destroy_orphaned_business_objective
     audit_bo_autoremove(removed_bos)
   end
 
+  def supported_os_minor_versions
+    profile_os_major_version = initial_profile.os_major_version
+    supported_benchmarks.map do |bm|
```

Comment:
Maybe you can get away with a `flat_map` and then you don't need the `flatten`. Not sure, please check...

```suggestion
    supported_benchmarks.flat_map do |bm|
```